<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>延时燃放</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="labelDelayTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>输入延时时间:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDelayTime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_s">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>10</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>25</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
